### PR TITLE
Add scopes

### LINKED ISSUE #993: Add Tags to rule (notably scopes)
We have a bit of tagging support to rules, scope should be added to it.
https://github.com/Siteimprove/alfa/blob/main/packages/alfa-rules/src/tags/scope.ts

### PR DESCRIPTION
**Pull request by Jym77**
Resolves #993 

I got need of scopes for some of the examples I want to write… Took the opportunity to clean up a bit the imports since I was touching every rule…

### CODE REVIEW

File: packages/alfa-rules/src/sia-r109/rule.ts [hunk 1]
```diff
@@ -24,8 +24,8 @@ const { and } = Refinement;
 export default Rule.Atomic.of<Page, Document, Question>({
   uri: "https://alfa.siteimprove.com/rules/sia-r109",
   requirements: [Criterion.of("3.1.1"), Technique.of("H57")],
-  tags: [Stability.Experimental],
-  evaluate({ device, document }) {
+  tags: [Scope.Page, Stability.Experimental],
+  evaluate({ document }) {
```
Comment (zsofitoth) [inline packages/alfa-rules/src/sia-r109/rule.ts:28]: How come the device is no longer needed?

File: packages/alfa-rules/src/sia-r75/rule.ts [hunk 2]
```diff
@@ -7,15 +7,16 @@ import { Page } from "@siteimprove/alfa-web";
...
 
-import { hasTextContent } from "../common/predicate/has-text-content";
-import { isVisible } from "../common/predicate/is-visible";
+import { hasTextContent, isVisible } from "../common/predicate";
+import { Scope } from "../tags";
 
 const { isElement, hasNamespace } = Element;
 const { isText } = Text;
-const { and, or, not } = Predicate;
+const { and, or } = Predicate;
```
Comment (zsofitoth) [inline packages/alfa-rules/src/sia-r75/rule.ts:15]: Was the `not` predicate unused?

**Review by zsofitoth — APPROVED** [review]
Looks good 👍 Just a few questions

**Comment by Jym77** [comment]
> Looks good 👍 Just a few questions

Apparently, they were both unused, but never cleaned up 🤷

**Pull request merged**